test/Wrappers/wrapper-erc20.test.ts

```diff
@@ -0,0 +1,72 @@
+import {ethers} from "hardhat"
+import { expect } from "chai"
+import { ERC20Wrapper } from "../../lib/wrappers/tokens/erc20"
+import { ERC20Mock } from "../../typechain-types"
+
+describe("ERC20 Wrapper", async () => {
+    let tokenContract: ERC20Mock
+
+    beforeEach(async () => {
+        const [deployer] = await ethers.getSigners()
+
+        tokenContract = await ethers.deployContract("ERC20Mock", [deployer, 100, "Mock", "MKT"])
+    })
```

Comment:
Directly create the wrapper here, if we need the contract you can use `wrapper.call` and if you need the contract address we can use `wrapper.address`

Reply:
Fixed in 118f2aa